content/courses/tokens-and-nfts/token-program-advanced.md

```diff
@@ -210,59 +210,109 @@ const delegate = new PublicKey("YOUR_DELEGATE_HERE");
 // Substitute in your token mint account
 const tokenMintAccount = new PublicKey("YOUR_TOKEN_MINT_ADDRESS_HERE");
 
-// Get or create the source and destination token accounts to store this token
-const sourceTokenAccount = await getOrCreateAssociatedTokenAccount(
-  connection,
-  user,
-  tokenMintAccount,
-  user.publicKey,
-);
+(async () => {
```

Comment:
You don't need to add this, in fact you were instructed to remove it: https://github.com/solana-foundation/developer-content/blob/main/CONTRIBUTING.md#jsts

Also add `filesname=` to the code snippets. See https://github.com/solana-foundation/developer-content/blob/main/CONTRIBUTING.md#file-names